lib/api/v1/user-controller.js

```diff
@@ -67,10 +72,12 @@ module.exports = function(app, user) {
   });
 
   // Update a User
-  app.put('/api/v1/user/:_id', user.can('manage users'), function(req, res) {
+  app.put('/api/v1/user/:_id', user.can('update users'), function(req, res) {
```

Comment:
Since all roles can update their own profile you perhaps don't need to protect this one at all, and just use your 'admin' check inside the action.